src/index.mako.html

```diff
@@ -751,8 +751,6 @@
           
           gaLayersProvider.terrainTileUrlTemplate = '//3d.geo.admin.ch/1.0.0/{Layer}/default/{Time}/4326';
 
-          gaLayersProvider.wmsMapProxyUrl = '//wmts{s}.geo.admin.ch/mapproxy/service';
-          
```

Comment:
Why not using a `gaLayersProvider.wmsUrl` here and removing it from layersConfig ?